calendar-be/PlannerApp/src/main/java/com/PlannerApp/PlannerApp/Models/Invite.java

```diff
@@ -0,0 +1,18 @@
+package com.PlannerApp.PlannerApp.Models;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.UUID;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Data
+@Builder
+public class Invite {
+    private UUID id;
+    private UUID user_id;
+    private UUID group_id;
```

Comment:
What is id used for and how its different from user_id?

Reply:
The ID will be used once we have a proper login system. A user who is in the group will want to send an invitation to this group. Therefore, they will enter another user’s ID (user_id) and invite them to the current group of the admin (group_id).